### PR TITLE
Added missing translations

### PR DESCRIPTION
**Pull request by cosmos-explorer**

### CODE REVIEW

File: modules/front-end/src/locale/messages.zh.xlf [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+<?xml version="1.0" encoding="UTF-8"?><xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
```
Comment (deleteLater) [inline modules/front-end/src/locale/messages.zh.xlf]: Why this change?

Comment (cosmos-explorer) [reply modules/front-end/src/locale/messages.zh.xlf]: reverted

**Review by deleteLater — APPROVED** [review]
LGTM
